.gitignore

```diff
@@ -1,4 +1,3 @@
-Dockerfile
```

Comment:
The deploy script is generating a Dockerfile as well, so this will conflict. I see you're using the Dockerfile for the tests in travis, so you can rename the file and make docker use a renamed one in travis

Reply:
The deploy script runs in the package repo, not here. I don't understand how this wil conflict.

Reply:
Oh right you have the package repo. Forgot about that, my fault.